source/rust_verify/src/main.rs

```diff
@@ -110,6 +110,27 @@ pub fn main() {
 
     let pervasive_path = our_args.pervasive_path.clone();
 
+    if our_args.error_report {
+        let mut args: Vec<String> = std::env::args().collect();
+        args.remove(0);
+
+        let index = args.iter().position(|x| *x == "--error-report").unwrap();
+        args.remove(index);
+
+        let verus_path = std::env::current_exe().unwrap();
+        let exe = verus_path.parent().unwrap().join("../../error_report/target/debug/error_report");
```

Comment:
This expects `error_report` to have been built with debug-only; might want to consider checking whether the debug or release builds are around, and pick the right one.